fv3core/stencils/remapping.py

```diff
@@ -20,6 +20,7 @@ def compute(
     va: FloatField,
     cappa: FloatField,
     q_con: FloatField,
+    q_cld: FloatField,
```

Comment:
Yeah, I changed it in this particular part as you said because q_cld isn't used in this routine in its capacity as a tracer, but rather is used in the called saturation adjustment routines. It's not even passed to `mapn_tracer`, which really confused me before I changed this.

We should keep these idiosyncracies in mind in the very soon to be enabled refactor of tracers. Probably will need a container object with all the tracers, which has attributes for advected tracers ("water" tracers?) and all tracers. When we get ready for that refactor we should write a doc with what it is we require (e.g. I think while we pass dicts to certain routines right now, a list/iterable of tracers is sufficient).

This behavior should be clearer mapping to and from the Fortran code when we refactor this into a data container.